specification/network/resource-manager/Microsoft.Network/stable/2021-08-01/networkSecurityGroup.json

```diff
@@ -870,6 +870,19 @@
     },
     "NetworkSecurityGroupPropertiesFormat": {
       "properties": {
+        "flushConnection": {
+          "type": "string",
+          "description": "When enabled, flows created from Network Security Group connections will be re-evaluated when rules are updates. Initial enablement will trigger re-evaluation.",
+          "enum": [
+            "true",
```

Comment:
>true [](http://example.com/codeflow?start=13&length=4)

this is not the intent of the "avoid booleans" linter rule.  "true"/"false" strings are not a good practice and confusing to customers.  Ideally this property is modeled as "connectionFlush": "Enabled"/"Disabled"

Reply:
I have updated the values to "Enabled" "Disabled" instead.

Reply:
I have reverted it back to boolean based on offline discussion.